src/components/myMeetings/Meeting.tsx

```diff
@@ -94,17 +94,20 @@ const Meeting: React.FC<
         <Heading as="h2" fontSize="1.5em">
           {title}
         </Heading>
-        <Text fontSize="1em">{formatMeetingTime(new Date(startTime))}</Text>
+        <Box minH="2rem">{description && <Text>{description}</Text>}</Box>
```

Comment:
This will show an empty div with 2rem even if description is missing. Is this intended? or should it be:

```react
{description && <Box minH="2rem"><Text>{description}</Text></Box>}
```